lib/matplotlib/textpath.py

```diff
@@ -287,9 +286,7 @@ def get_glyphs_mathtext(self, prop, s, glyph_map=None,
                 glyph_map_new, myrects)
 
     def get_texmanager(self):
-        """
-        return the :class:`matplotlib.texmanager.TexManager` instance
-        """
+        """Return a cached `~.texmanager.TexManager` instance."""
```

Comment:
```suggestion
        """Return the cached `~.texmanager.TexManager` instance."""
```

There is just one instance.